script/sync_prod_to_gcp/test/ntai.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/bash
```

Comment:
Please don't check in a file named "ntai.sh" Please give it a more descriptive name.  Or don't check it in since there is script/sync_prod_to_gcp/submissions-to-gcp.sh ?

Reply:
Done.

Reply:
It was to use the arxiv-development queue and bucket. Renamed it accordingly.